### ISSUE TITLE
Creating new canvas gives 500 error

### ISSUE THREAD

**Issue by bobby-mccann** [darklang/dark#4575]
In settings it says to create a new canvas by navigating to the URL:
![image](https://user-images.githubusercontent.com/88658699/199533627-89be4d1c-223f-421b-a466-b29f3b7f6b8c.png)

But if I go to a url (eg. https://darklang.com/a/new-canvas) I get a 500 error.

**Comment by StachuDotNet** [comment]
You need to go to a URL like `https://darklang.com/a/bogoblin-new-canvas`

We could certainly make that more clear!

**Comment by bobby-mccann** [comment]
Want me to fix this?

**Comment by StachuDotNet** [comment]
Yes, please go ahead! If you need any assistance, happy to chat here or Discord.

**Comment by pbiggar** [comment]
Or could add a text input and a button to open the new canvas

**Comment by pbiggar** [comment]
To clarify, something like this:

<pre>
Create a new canvas: <i>username-</i> [__________] [Create]
</pre>

**Comment by StachuDotNet** [comment]
@bobby-mccann Are you still interested in tackling this? We'd be happy to assist you in getting set up, and I'd be happy to pair on the work itself if you'd like.

**Comment by pbiggar** [comment]
Oh, this was actually done in #4601
